hi-ml-histopathology/Makefile

```diff
@@ -57,3 +57,7 @@ pytest:
 # run pytest with coverage on package
 pytest_coverage:
 	pytest --cov=histopathology --cov SSL --cov-branch --cov-report=html --cov-report=xml --cov-report=term-missing --cov-config=.coveragerc
+
+# Run regression tests and compare performance
+run_regression_test:
+	cd ../; python hi-ml/src/health_ml/runner.py --model=histopathology.TcgaCrckImageNetMIL --crossval_count=1 --additional_env_file=hi-ml-histopathology/environment.yml --pl_limit_train_batch=2 --pl_limit_val_batch=2 --pl_limit_test_batch=2 --max_bag_size=3 --max_bag_size_inf=3 --max_epochs=2 --azureml --cluster=innereye4cl --regression_test_folder=hi-ml-histopathology/RegressionTestResults/HD_13448836-ddc9-4000-87a4-f84075d9d29c_1 --regression_test_csv_tolerance=0.05
```

Comment:
Note that when we have decided on the proper contents of RegressionTestResults, these limiting flags will be removed and the full-length experiment will be run